src/css/skills.scss

```diff
@@ -0,0 +1,140 @@
+@import './core/constants';
+@import './core/mixins';
+
+#skills {
+  min-height: 708.3px;
+  background-image: url('./img/background.jpg');
+  background-attachment: fixed;
+  position: relative;
+
+  .skills-section {
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -41%);
+
+    h2 {
+      color: $white;
+      font-size: $font-xl;
+    }
+
+    #skills-hr {
+      background: $white;
+    }
+
+    .skills-ul {
+      display: flex;
+      flex-wrap: wrap;
+      float: top;
+
+      li {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+      }
+    }
+
+    #skills-detail-container {
+      height: 434px;
+      width: 942px;
+      background-color: greenyellow;
+      font-family: $font-family-title;
+      font-weight: bold;
+
+      #skills-li-excel {
+        height: 145.5px;
+        width: 145.5px;
+        background-color: $light-cyan;
+
+        & div:first-child {
+          font-size: 4px;
+          -webkit-transform: scale(0.7);
```

Comment:
I don't think you need to scale anything here, just use different font sizes.
scale sometimes make thing looks fuzzy.
also why do you only targeting webkit browsers with `-webkit-` prefix?